framework/db/Command.php

```diff
@@ -90,6 +90,11 @@ class Command extends Component
      */
     private $_pendingParams = [];
     /**
+     * @var mixed INSERT/UPDATE return parameters to be bound to the current PDO statement
+     * or boolean true if return parameters are the query result.
```

Comment:
In `Command::queryInternal`. The comment here is rather unclear. I didn't want to introduce another argument to the method so I just used this as a special value.

There are other quirks, all the steps necessary to call it are not intuitive:

* create a command using insertReturning
* prepare for writing using prepare(false)
* don't just execute, use queryOne